crash-handler-process/logger.cpp

```diff
@@ -24,22 +25,27 @@ std::ofstream log_output_file;
 const std::string getTimeStamp() 
 {
     const std::time_t t = std::time(nullptr);
+
+    struct tm buf;
+    localtime_s(&buf, &t);
+
     char mbstr[128]={0};
-    std::strftime(mbstr, sizeof(mbstr), "%Y%m%d:%H%M%S.", std::localtime(&t));
+    std::strftime(mbstr, sizeof(mbstr), "%Y%m%d:%H%M%S.", &buf);
     unsigned __int64 now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
-    std::ostringstream ss;
+    
+	std::ostringstream ss;

```

Comment:
Indentation here is different

Reply:
Thanks. I missed to update it. 
Replaced with TAB  indentation for all file as in other files.